velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -56,12 +56,34 @@ class Re2FunctionsTest : public test::FunctionBaseTest {
   }
 
  protected:
+  const char* alphaNumCharSet_ =
+      "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  const char* wildcardCharSet_ = "_%";
+  const char* singleWildcardCharSet_ = "_";
+  const char* anyWildcardCharSet_ = "%";
+
   template <typename T>
   void testRe2ExtractAll(
       const std::vector<std::optional<std::string>>& inputs,
       const std::vector<std::optional<std::string>>& patterns,
       const std::vector<std::optional<T>>& groupIds,
       const std::vector<std::optional<std::vector<std::string>>>& output);
+
+  std::string generateRandomString(
+      const char* charArr,
+      std::optional<vector_size_t> minLen = 1,
```

Comment:
These arguments don't need optional.

```
vector_size_t minSize = 1,
vector_size_t maxSize = 10
```

charArr argument name is not very descriptive. Consider characterSet.